app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# User for our fakebook app
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  before_action :authenticate_user!
```

Comment:
This line generates the following error:
```
`method_missing': undefined method `before_action' for User (call 'User.connection' to establish a connection):Class (NoMethodError)
Did you mean?  before_commit
```

Reply:
yes we are working on this issue. I submitted for review before we fixed this. will resubmitt once we have this fixed correctly.